README.md

```diff
@@ -65,19 +67,47 @@ The Checkout JS SDK is the easiest way to build a bespoke checkout into your sto
 
 ## Installation
 
+### Using NPM package
+
 You can install this library using [npm](https://www.npmjs.com/get-npm).
 
 ```sh
 npm install --save @bigcommerce/checkout-sdk
 ```
 
+### Using CDN URL
+
+You can also use this library by referencing a CDN URL.
+
+```
+https://checkout-sdk.bigcommerce.com/v1/loader.js
+```
+
+The main benefit of using the script URL above is that your application can automatically receive backward compatible updates and bug fixes from us, without having to manually perform an upgrade.
+
+Once the above script is loaded, you can use it to load the module that you need for your application. i.e.:
+
+```js
+const module = await checkoutKitLoader.load('checkout-sdk');
```

Comment:
I'm not very familiar with this approach. Would typescript still work using the CDN approach? is `checkoutKitLoader` a global object that becomes available when referencing `https://checkout-sdk.bigcommerce.com/v1/loader.js` ?

Reply:
So this is JS only. For TS, we'll need to figure out something in the future so that we can provide types statically and the code dynamically.

Regarding your second point on `checkoutKitLoader`, yes you're correct.